tasks/main.yml

```diff
@@ -47,8 +47,13 @@
 
 - name: Set a fact containing paths to DIB elements
   set_fact:
+    os_images_elements_path: "{{ os_images_elements }}"
+  when: os_images_elements_path is undefined
```

Comment:
I don't think the when clause is required here - os_images_elements_path isn't part of the interface to the role, it's just a working variable. If only ansible had a better notion of scope.